src/components/InputBox/InputBox.jsx

```diff
@@ -0,0 +1,14 @@
+import './InputBox.scss';
+
+const InputBox = ({title , inputType, errorMessage, successMessage}) => {
+    return(
+        <div className='input-container'>
```

Comment:
- classname should be 'input' rather than 'input-container' and follow on with BEM